apisix/plugins/consumer-restriction.lua

```diff
@@ -51,7 +51,8 @@ local schema = {
                 }
             }
         },
-        rejected_code = {type = "integer", minimum = 200, default = 403}
+        rejected_code = {type = "integer", minimum = 200, default = 403},
+        rejected_message = {type = "string"}
```

Comment:
```suggestion
        rejected_msg = {type = "string"}
```
Would it be better to change `rejected_message` to `rejected_msg`? to keep it similar to other plugins.

Reply:
ok